helm/macropower-analytics-panel-server/templates/ingress-auth.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "resource.default.name" . }}-auth
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  rules:
+  - host: {{ .Values.server.ingress.host }}
+    http:
+      paths:
+      - backend:
+          service:
+            #TODO: template name and port number
+            name: grafana
+            port:
+              number: 3000
+        #TODO: change path
+        path: /grafana-fauth
```

Comment:
grafana-fauth ?

Reply:
```suggestion
        path: /grafana-auth
```

Reply:
I do not know what to put in there yet, `/grafana-auth`, `/analytics-plugin/auth` ? unsure yet

Reply:
If it's only for the plugin, I would go with /analytics-plugin/auth or login yes :)